shell_completions.md

```diff
@@ -82,6 +111,8 @@ PowerShell:
 			cmd.Root().GenFishCompletion(os.Stdout, true)
 		case "powershell":
 			cmd.Root().GenPowerShellCompletionWithDesc(os.Stdout)
+		case "nushell":
+			cmd.Root().GenNushellCompletion(os.Stdout, true)
```

Comment:
Since we have the cobra_apps array now, there is one way could add support on a per app basis by doing something like this:

```
let cobra_apps = [
  {app: 'kubectl', include_help: true},
  {app: 'helm', include_help: false},
]
```

Reply:
@marckhouzam, thoughts?

Reply:
Will need to give it some thought. Will get back to the review over the weekend.  Thanks @ayax79